plugins/tutor-contrib-paragon/README.rst

```diff
@@ -0,0 +1,63 @@
+Paragon plugin for `Tutor <https://docs.tutor.edly.io>`__
+###########################################################
+
+Facilitates the generation and static hosting of Paragon-based CSS themes for Open edX Micro-Frontend (MFE) applications using `Paragon <https://openedx.github.io/paragon/>`__.
+
+This plugin provides a local folder structure to manage theme source files and compile them into CSS, enabling flexible customization of Open edX MFEs via Tutor.
+
+Installation
+************
+
+.. code-block:: bash
+
+    pip install git+https://github.com/openedx/openedx-tutor-plugins.git#subdirectory=plugins/tutor-contrib-paragon
+
+For development:
+
+.. code-block:: bash
+
+    cd openedx-tutor-plugins/plugins/tutor-contrib-paragon
+    pip install -e .
+
+Enable the plugin:
+
+.. code-block:: bash
+
+    tutor plugins enable paragon
+
+Directory Structure
+*******************
+
+The plugin will create the following structure inside your Tutor environment:
+
+.. code-block::
+
+    tutor/env/plugins/paragon/
+    ├── theme-sources/           # Place your style dictionary folders here (e.g. theme-xyz/)
+    └── compiled-themes/         # Compiled CSS files go here, ready for static hosting
+
+Each theme inside `theme-sources/` should follow the Style Dictionary folder structure.
```

Comment:
It's not clear from this line what the "Style Dictionary folder structure" is. A link explaining that would do wonders for clarity.